mythril/analysis/modules/dependence_on_predictable_vars.py

```diff
@@ -159,19 +164,20 @@ def _analyze_states(state: GlobalState) -> list:
 
                     index = r.group(1)
                     if index and solve(call):
-                        description += (
-                            "block.blockhash() is calculated using a value from storage "
+                        description = (
+                            "Block.blockhash() is calculated using a value from storage "
```

Comment:
When starting a sentence with a segment of code, is it correct to capitalize the code? Probably add a word or two in front of `block.blockhash()`.